File: py110/small_problems_110_119/debugging/1.py
# https://launchschool.com/exercises/267890f0?track=python

def decrease(counter):
    return counter - 1

counter = 10

for _ in range(10):
    print(counter)
    counter = decrease(counter)

print('LAUNCH!')



# counter = 10

# for _ in range(10):
#     print(counter)
#     counter -= 1

# print('LAUNCH!')

File: ls215/smallProblems/listProcessing/1.js
// https://launchschool.com/exercises/a887cdb7?track=python

function sum(number) {
  return String(number)
    .split('')
    .reduce(((total, num) =>  total + Number(num)), 0);
}

sum(23);           // 5
sum(496);          // 19
sum(123456789);    // 45